aiohttp/web_ws.py

```diff
@@ -431,23 +431,10 @@ async def close(
         if self._writer is None:
             raise RuntimeError("Call .prepare() first")
 
-        self._cancel_heartbeat()
```

Comment:
Should this line have gone? Just double checking.

Reply:
Yes, `_set_closed()` handles it already https://github.com/aio-libs/aiohttp/blob/bbe71972b4452cd45ed94f5546722a31860a5dda/aiohttp/web_ws.py#L209